sentry/src/main/java/io/sentry/SentryAttributes.java

```diff
@@ -0,0 +1,44 @@
+package io.sentry;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryAttributes {
+
+  private final @NotNull List<SentryAttribute> attributes;
```

Comment:
should we change this to Map<String, SentryAttribute>?
Now it's possible to add multiple times the same attribute, but only the last one will actually be sent